pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -167,6 +174,66 @@ func (og *operationGenerator) GenerateVolumesAreAttachedFunc(
 	}, nil
 }
 
+func (og *operationGenerator) GenerateBulkVolumeVerifyFunc(
```

Comment:
I would be so much happier if we didn't have two parallel functions, though I realize that is tricky.

One option _could_ be to return an `[]Operation`, which would include the key & the function.